core/src/main/scala/org/broadinstitute/dsde/rawls/expressions/parser/antlr/DataRepoInputExpressionValidationVisitor.scala

```diff
@@ -22,19 +24,23 @@ class DataRepoInputExpressionValidationVisitor(rootEntityType: Option[String],
   override def visitEntityLookup(ctx: EntityLookupContext): Try[Unit] = {
     rootEntityType match {
       case Some(rootTableName) => validateEntityLookup(rootTableName, ctx)
-      case None => Failure(new RawlsException("Expressions beginning with \"this.\" are only allowed when running with workspace data model. However, workspace attributes can be used."))
+      case None => Failure(new RawlsExceptionWithErrorReport(
+        ErrorReport(StatusCodes.BadRequest, "Expressions beginning with \"this.\" are only allowed when running with workspace data model. However, workspace attributes can be used.")
```

Comment:
// it seems like it would get here only if methodConfiguration.rootEntityType is not set, so this error message doesn't make sense to me. Shouldn't it be something like "Missing Root Entity Type. Please select one and try again."

Reply:
slack conversation about this: https://broadinstitute.slack.com/archives/GSHUGJ5RD/p1596826199127100

I'm okay with leaving it for now. It seemed off to me so I just wanted to make sure it was ok